sdk/js/src/examples/simpleSearcherLimo.ts

```diff
@@ -142,6 +144,12 @@ class SimpleSearcherLimo {
       order.state.globalConfig
     );
     const bidAmount = new anchor.BN(argv.bid);
+    if (this.bidMargin !== 0) {
+      const margin = new anchor.BN(
+        Math.floor(Math.random() * (this.bidMargin * 2 + 1)) - this.bidMargin
+      );
+      bidAmount.add(margin);
```

Comment:
this doesn't change the bidAmount. I think you should do
`bidAmount = bidAmount.add(margin)`